aksel.nav.no/website/pages/god-praksis/[slug].tsx

```diff
@@ -67,7 +68,7 @@ export const getServerSideProps: GetServerSideProps = async (
       articles: await Promise.all(
         articles.map(async (a) => ({
           ...a,
-          publishedAt: await dateStr(a.publishedAt),
+          lastVerified: await dateStr(a.lastVerified ?? a.publishedAt),
```

Comment:
Consider if this derived value should have a different name, e.g. "displayDate" or just "date". Not a big deal, but it might be confusing for developers when values are changed (and doesn't correspond with the value with the same name in the db).

Reply:
Went for `displayDate` 👍